### DISCUSSION TITLE
remove case from match and use _ instead default

### DISCUSSION THREAD

**Discussion by igotfr** [carbon-language/carbon-lang#1672] [Language design]
```rust
match (Bar()) {
    (42, (x: f32, y: f32)) => { return x - y; },
    (p: i32, (x: f32, _: f32)) if (p < 13) => { return p * x; },
    (p: i32, _: auto) if (p > 3) => { return p * Pi; },
    _ => { return Pi; }
}
```

**Comment by YagaoDirac** [comment]
Is this global function overload? Or equivalence to switch in c++?
But for both cases, I'm sorry I don't agree.
More grammar means less regularity. Less regularity means weaker auto completion.

**Comment by OlaFosheimGrostad** [comment]
I also have a preference for the symbol _ generally meaning: ignore, match anything, fill-in-the-blank etc. I would like to see this being used, and used consistently.
